.github/workflows/frontend-check.yaml

```diff
@@ -1,51 +1,76 @@
-name: Cypress tests
+name: Cypress end-to-end tests
 on: 
   push:
     branches:
       - main
       - revamp # rm when branch is deleted
   pull_request:
 jobs:
-  install:
-    runs-on: ubuntu-24.04
+  cypress-run-ci:
+    runs-on: ubuntu-latest
     steps:
+      # - name: install podman
+      #   run: apt-get -y install podman
```

Comment:
Let's not check in commented "code" -- at least not without a good reason and an explanatory comment.

Reply:
removed